db/config.js

```diff
@@ -0,0 +1,16 @@
+require('dotenv').config()
+
+const { Pool } = require('pg')
+
+const isProduction = process.env.NODE_ENV === 'production'
```

Comment:
Hmm, I see what you're saying, but am not sure how to set it up in this config. That solution passes the request along each time,  but we're checking this in the initial pool setup. I imagine this would have to be done at the query level then somehow, and not during pool setup. I'll take this out for now.

Reply:
Right, gotcha. Yeah, let's leave it out then.